srcpkgs/snes9x/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'snes9x'
+pkgname=snes9x
+version=1.56.2
+revision=1
+build_wrksrc="unix"
+build_style=gnu-configure
+configure_args="--enable-netplay"
+hostmakedepends="intltool nasm autogen autoconf automake"
+makedepends="minizip-devel pulseaudio-devel libpng-devel SDL-devel SDL2-devel libXv-devel
+ xorg-server-devel zlib-devel gtk+3-devel"
+depends="minizip libpulseaudio pulseaudio desktop-file-utils libpng SDL SDL2 adwaita-icon-theme
```

Comment:
don't need to add stuff that is detected via shlibs

Reply:
Which specifically is detect via shlibs? Sorry this is first day I have done contributions